charts/ocis/README.md

```diff
@@ -38,80 +39,281 @@ The command removes all the Kubernetes components associated with the chart and
+
+  # how to generate: base64 encode a x509 certificate signed by the above CA, using the above private key.
+  # example generation commands:
+  #  - `openssl req -new -subj "/CN=idm" -key ldap.key -out ldap.csr`
+  #  - `openssl x509 -req -extensions SAN -extfile <(cat /etc/ssl/openssl.cnf <(printf "\n[SAN]\nsubjectAltName=DNS:idm")) -days 365 -in ldap.csr -CA ldap-ca.crt -CAkey ldap-ca.key -out ldap.crt -CAcreateserial`
+  #  - `cat ldap.crt | base64 | tr -d '\n'`
+  ldap.crt: XXXXXXXXXXXXX
+
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: machine-auth-api-key
+type: Opaque
+data:
+  # how to generate: base64 encode a random string (reasonable long and mixed characters)
+  # example generation command: `tr -cd '[:alnum:],.' < /dev/urandom | fold -w 50 | head -n 1 | base64`
+  machine-auth-api-key: XXXXXXXXXXXXX
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: storage-system
+type: Opaque
+data:
+  # how to generate: base64 encode a UUID V4
+  # example generation command: `cat /proc/sys/kernel/random/uuid | tr -d '\n' | base64`
+  user-id: XXXXXXXXXXXXX
+
+  # how to generate: base64 encode a random string (reasonable long and mixed characters)
+  # example generation command: `tr -cd '[:alnum:],.' < /dev/urandom | fold -w 50 | head -n 1 | base64`
+  api-key: XXXXXXXXXXXXX
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: storage-system-jwt-secret
+type: Opaque
+data:
+  # how to generate: base64 encode a random string (reasonable long and mixed characters)
+  # example generation command: `tr -cd '[:alnum:],.' < /dev/urandom | fold -w 50 | head -n 1 | base64`
+  storage-system-jwt-secret: XXXXXXXXXXXXX
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: transfer-secret
+type: Opaque
+data:
+  # how to generate: base64 encode a random string (reasonable long and mixed characters)
+  # example generation command: `tr -cd '[:alnum:],.' < /dev/urandom | fold -w 50 | head -n 1 | base64`
+  transfer-secret: XXXXXXXXXXXXX
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: admin-user
+type: Opaque
+data:
+  # how to generate: base64 encode a UUID V4
+  # example generation command: `cat /proc/sys/kernel/random/uuid | tr -d '\n' | base64`
+  user-id: XXXXXXXXXXXXX
+
+  # how to generate: base64 encode a random string (reasonable long and mixed characters)
+  # example generation command: `tr -cd '[a-zA-Z0-9],.' < /dev/urandom | fold -w 50 | head -n 1 | tr -d '\n' | base64`
+  password: XXXXXXXXXXXXX
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: idp-secrets
+type: Opaque
+data:
+  # how to generate: base64 encode a random 32 byte string (mixed characters)
+  # example generation command: `openssl rand 32 | base64`
+  encryption.key: XXXXXXXXXXXXX
+
+  # how to generate: base64 encode a private key (eg. RSA, ensure that you use reasonable long key size)
+  # example generation command: `openssl genpkey -algorithm RSA -pkeyopt rsa_keygen_bits:4096 2&> /dev/null | base64 | tr -d '\n'`
```

Comment:
It think you want `2> /dev/null` here instead of `2&> /dev/null` , don't you?

Reply:
thanks, good catch!